<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitWindow</class>
 <widget class="QMainWindow" name="FitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AGEFitViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="layoutMain" rowstretch="1">
    <item row="0" column="0" colspan="3">
     <widget class="QTabWidget" name="tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabFitting">
       <attribute name="title">
        <string>Interactive Fitting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupParams">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Parameters</string>
          </property>
          <layout class="QVBoxLayout" name="layoutParameters">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>751</width>
                <height>733</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="layoutParams"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupFitView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="title">
           <string>FitView</string>
          </property>
          <layout class="QVBoxLayout" name="layoutFitView"/>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupFitSetup">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="layoutFitSetup">
           <item>
            <widget class="QGroupBox" name="groupCost">
             <property name="title">
              <string>Cost Function</string>
             </property>
             <layout class="QHBoxLayout" name="layoutCost">
              <item>
               <widget class="QComboBox" name="selectCost"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupModel">
             <property name="title">
              <string>Model</string>
             </property>
             <layout class="QHBoxLayout" name="layoutModel"/>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupFit">
             <property name="title">
              <string>Fit</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="buttonFit">
                <property name="text">
                 <string>Minimize</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabResults">
       <attribute name="title">
        <string>Results</string>
       </attribute>
       <layout class="QVBoxLayout" name="layoutResults">
        <item>
         <widget class="QPlainTextEdit" name="textResults">
          <property name="font">
           <font>
            <family>FreeMono</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumatplotlib">
    <property name="title">
     <string>matplotlib</string>
    </property>
    <addaction name="actionSetLabels"/>
    <addaction name="actionSetTitle"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <addaction name="menumatplotlib"/>
  </widget>
  <action name="actionSetLabels">
   <property name="text">
    <string>Set Axes Labels</string>
   </property>
  </action>
  <action name="actionSetTitle">
   <property name="text">
    <string>Set Axes Title</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save Figure As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
